### ISSUE TITLE
Discovery: Conduct User Interviews

### ISSUE THREAD

**Issue by rileyorr** [department-of-veterans-affairs/va.gov-team#3424]
## User Story or Problem Statement
As a UX Designer, I need conduct interviews with Veterans who order batteries and/or prosthetic socks so I can find pain points within the ordering system and develop empathy with our users.

## Goal
_Conduct up to 16 (8) interviews with Veterans who order batteries and/or prosthetic socks from the VA/DLC_

## Objectives or Key Results this is meant to further
- _Collect data about how Veterans order batteries and/or prosthetic socks_
- _Find pain points in the ordering process_
- _Fill gaps within the ordering process_
- _Understand the ordering process_
- _Influences the form's flow and design_
- _Develop empathy with our users_

## Tasks
- [x] Assign Facilitators and Notetakers for each session_ (1 hour)
- [x] Conduct a practice interview session with the team_ (1.5 hours)
- [x] Conduct the interviews scheduled by Perigean (18-24 hours for each designer)
- [x] Ensure the notes document is complete with the team's notes_ (9 hours)
- [x] Create conversation guide to speak with DLC Customer Service Representatives (2 hours)
- [x] Share convo guide with team for feedback (30 min) 

## Acceptance Criteria
- [x] Interviews completed
- [x] Notes in GH [(Link to folder)](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/medical-device-tool/research/discovery-nov19/notes)
- [x] Conversation guide created and uploaded to GH [Link](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/medical-device-tool/research/discovery-nov19/dlc-representative-conversation-guide.md)

## Next Steps
- Synthesis

## Blockers
- Participant scheduling issues
- DLC scheduling issues

**Comment by rebeccawalsh** [comment]
This ticket was originally sized at a 5 in anticipation of only about 8 participants showing up (50%), we experienced more like 90% participation. We had 11 total participants, 10 of which were qualified hearing aid participants. We cancelled the remaining HA-only participants due to diminishing returns. All participants who indicated socks were not qualified participants, leading us to extend our study to the end of the sprint after providing Perigean with feedback. Perigean was unable to source appropriate participants within the sprint.

@rileyorr  and I decided that, in order to learn more about the sock users, we would speak to customer service representatives knowledgeable in the subject for secondary research. We hoped to still source some sock participants to validate or expand upon our CS interview findings. 

At @wygonikGCIO  direction, we are holding off on extending our study to gather qualified sock participants due to low sock order volume.
